database/mysql/mysql_db.py

```diff
@@ -57,6 +57,11 @@
         files (C(.sql)) as well as bzip2 (C(.bz2)), gzip (C(.gz)) and xz (Added in 2.0) compressed files are supported.
     required: false
 author: "Ansible Core Team"
+requirements:
+   - mysql
+   - mysqldump
```

Comment:
normally it is assume that these are python libs, i would recommend stating that they are the command line utilities.